react/react1/week1/todo-list/src/ListToDo.js

```diff
@@ -0,0 +1,15 @@
+import { ActionAttributes } from "./ActionAttributes";
+
```

Comment:
We I think we need a different name than ActionAttributes, imo. Thats not entirely meaningful.
Also the name hellou ?
As mentioned, I would prefer not to inline the map operation in the JSX.

Reply:
yes I will change the names, "hellou" is absolutely not meaningful I was playing around props  😳😳😳